### DISCUSSION TITLE
Viewport addon to render multiple sizes at once

### DISCUSSION THREAD

**Discussion by itaditya** [storybookjs/storybook#25350] [Ideas]
**Is your feature request related to a problem? Please describe**
When updating component styles many time it gets fixed in one viewport but breaks in another. After every change, switching between viewports to verify everything is tedious.

**Describe the solution you'd like**
Let user select multiple sizes in the Viewport Addon menu so the component rendered in Canvas can be worked upon considering all important breakpoints.

This problem becomes even more exemplified when Container Queries enter the picture. Since that brings responsiveness at component level instead of window level, devs would want to see whether their design system component can work across different container sizes and not just discrete window breakpoints.

**Describe alternatives you've considered**
Open Storybook in multiple windows with different viewport values and stack them side by side. Although working on single screen with editor open is very problematic and multiple browser windows would increase CPU load as well.

**Are you able to assist to bring the feature to reality?**
yes, I can...

**Additional context**

**Comment by IonelLupu** [comment]
this would be an amazing feature

**Comment by oleksii-lukin** [comment]
This addon looks like what you wanted. But it seem to be for an older version though.

https://github.com/vizeat/storybook-addon-responsive-views
